doc/supported-platforms.md

```diff
@@ -11,6 +11,7 @@ Ignition is currently only supported for the following platforms:
 * [Packet] - Ignition will read its configuration from the instance userdata. SSH keys are handled by coreos-metadata.
 * [QEMU] - Ignition will read its configuration from the 'opt/com.coreos/config' key on the QEMU Firmware Configuration Device.
 * [DigitalOcean] - Ignition will read its configuration from the droplet userdata. SSH keys and network configuration are handled by coreos-metadata.
+* [Hetzner Cloud] - Ignition will read its configuration from the instance userdata. SSH keys are handled by coreos-metadata.
```

Comment:
This is missing the actual link for "Hetzner Cloud". Also, I think we can drop the ssh keys part.

Reply:
Indeed, thanks, missed that! 

There currently is no "booting-on-hcloud" document yet in the `coreos/docs` repo yet. Is it fine if we add this later and change the docs here to point to that place or would you prefer to simply not mention it in the Supported Platforms section as long as there is no clear guide on booting a CoreOS instance in the Hetzner Cloud?